gui/model/falsecolor/falsecoloringcacheitem.h

```diff
@@ -4,16 +4,19 @@
 /** Cache item for computed false color images. */
 class FalseColoringCacheItem {
 public:
-	FalseColoringCacheItem() : pixmap_() {}  // invalid cache item
-	FalseColoringCacheItem(QPixmap img) : pixmap_(img)  {} // valid cache item
+	FalseColoringCacheItem() : pixmap_(), mat_(nullptr) {}  // invalid cache item
+	FalseColoringCacheItem(QPixmap img, cv::Mat3f mat) : pixmap_(img), mat_(new cv::Mat3f(mat))  {} // valid cache item
 
 	// Default copy constructor and assignment operator.
 
 	void invalidate() { pixmap_ = QPixmap(); }
 	bool valid() { return ! pixmap_.isNull(); }
 	QPixmap pixmap() { return pixmap_; }
+	cv::Mat3f* mat() { return mat_; }
+
 private:
 	QPixmap pixmap_;
+	cv::Mat3f* mat_;
```

Comment:
same as in distviewmodel. don't use a pointer. also I don't see a delete anywhere!?